wiki/Music_theory/Time_signature/en.md

```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+tags:
+  - music theory
+  - mapping
+  - beatmapping
+  - tips
+  - sheet music
+---
+
+# Time signature
+
+*For more information on time signatures in music theory, see: [Time signature](https://en.wikipedia.org/wiki/Time_signature)*
+
+A **time signature** is a musical term that indicates the number of [beats](/wiki/Beat) in a single measure (indicated by the top number), and which note indicates a single beat (indicated by the bottom number).
```

Comment:
- too much "indicate"s (I counted 4)
- you use terms such as "top number" and "bottom number", but before that there is nothing about what a time signature actually looks like

Reply:
i didnt write the article